prep-javascript-conditionals/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Conditionals</title>
+</head>
+
+<body>
+  <script>
+    // Your code here
+    var age = 22;
+    if (age < 18) {
+      console.log('a Child');
+    } else {
+      console.log('an Adult');
+    }
+    var student1Score = 50;
+    if (student1Score >= 70) {
+      console.log('Congrats! You Received a Passing Grade!');
+    } else {
+      console.log('Oh no you Failed!');
+    }
+    var student2Score = 95;
+    if (student2Score < 60) {
+      console.log('F');
+    }
+    if (student2Score < 70) {
+      console.log('D');
+    }
+    if (student2Score < 80) {
+      console.log('C');
+    }
+    if (student2Score < 90) {
+      console.log('B');
+    }
+    if (student2Score < 100) {
+      console.log('A');
+    } else {
+      console.log('A++');
+    }
```

Comment:
This should be a series of `if else if` statements

Reply:
Updated!

<img width="1440" alt="Screen Shot 2021-09-24 at 10 41 17 AM" src="https://user-images.githubusercontent.com/88061673/134693585-91975234-1ad0-4537-a5f2-dea8a400a966.png">